documentation/src/main/asciidoc/public/reference/_limitations.adoc

```diff
@@ -2,7 +2,7 @@
 = [[elasticsearch-limitations]] Known issues and limitations
 
 [[limitations-parallel-embedded-update]]
-== Without coordination, in rare cases, automatic indexing involving `@IndexedEmbedded` may lead to out-of sync indexes
+== Without coordination, in rare cases, indexing involving `@IndexedEmbedded` may lead to out-of sync indexes
```

Comment:
This is a bit misleading as mass indexing alone cannot cause this problem... but at the same time I agree that "explicit or listener-triggered indexing" is a mouthful.

I suppose we could use something like "targeted indexing" or "planned indexing" instead of "explicit or listener-triggered indexing" and as opposed to "mass indexing"/"JBatch indexing", but I'm not sure it's very clear. Maybe with a link to that section where you explain explicit or listener-triggered indexing... But that's not possible in headings :/

Well, it's a problem for another day, I guess.

Reply:
I'll add a todo next to this title, so it'll be more "visible" in the future 😃